packages/scene-composer/src/store/slices/SceneDocumentSlice.ts

```diff
@@ -326,6 +328,17 @@ export const createSceneDocumentSlice = (set: SetState<RootState>, get: GetState
         }
         draft.lastOperation = 'updateComponentInternal';
       });
+
+      const updatedComponenet = get().document?.nodeMap[nodeRef]?.components[componentToUpdateIndex];
+      const layerId = get().getSceneProperty<string[]>(KnownSceneProperty.LayerIds)?.at(0);
+      if (
+        (componentType === KnownComponentType.Tag ||
+          componentType === KnownComponentType.DataOverlay ||
+          componentType === KnownComponentType.EntityBinding) &&
```

Comment:
don't need to check component type. although we don't support other types yet, the updateEntity should just ignore other component types, so when we support other types, we don't need to go back to here and update

Reply:
removed